### PR TITLE
Publish error

### PR DESCRIPTION
**Pull request by alberto-baron**

### CODE REVIEW

File: src/PackageUploader.Application/Operations/PublishPackagesOperation.cs [hunk 1]
```diff
@@ -51,16 +51,24 @@ protected override async Task ProcessAsync(CancellationToken ct)
         }
 
         // Log validation errors if any
```
Comment (lawhelpt) [inline src/PackageUploader.Application/Operations/PublishPackagesOperation.cs:53]: Probably don't need the comments here

File: src/PackageUploader.Application/Operations/PublishPackagesOperation.cs [hunk 2]
```diff
@@ -51,16 +51,24 @@ protected override async Task ProcessAsync(CancellationToken ct)
...
+                    validationFailed = true;
+                    _logger.Log(GetLogLevel(validationItem.Severity), "{validationMessage}", validationItem.Message);
                 }
             });
         }
+
+        // Throw error if validation failed
+        if (validationFailed)
+        {
+            throw new Exception("Submission Validation Failed");
```
Comment (lawhelpt) [inline src/PackageUploader.Application/Operations/PublishPackagesOperation.cs:70]: So at this point, we have completed uploading the package right?

Comment (lawhelpt) [reply src/PackageUploader.Application/Operations/PublishPackagesOperation.cs:70]: Ahhh this is on the Publish operation, so yes it is part of the later process when someone is running this.

Comment (alberto-baron) [reply src/PackageUploader.Application/Operations/PublishPackagesOperation.cs:68]: yup :)

**Comment by lawhelpt** [comment]
Branch is 5 commits behind main...?

**Comment by alberto-baron** [comment]
> Branch is 5 commits behind main...?

True! Should be fixed now :)

**Review by lawhelpt — APPROVED** [review]

**Pull request merged**